clusters/hive/pools/openshift-ci/ci-ocp-4-10-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    region: us-east-1
+    version: "4.10"
+    version_lower: 4.10.0-0
+    version_upper: 4.10.0-0
```

Comment:
is `4.10.0-0` correct? looking at the other ClusterPools I would expect `4.11.0-0`

Reply:
Well spotted